core/capabilities/syncer.go

```diff
@@ -543,7 +543,12 @@ func (s *registrySyncer) addReceiver(ctx context.Context, capability kcr.Capabil
 
 	s.lggr.Debugw("Enabling external access for capability", "id", fullCapID, "donID", don.Id)
 	err = s.dispatcher.SetReceiver(fullCapID, fmt.Sprint(don.Id), receiver)
-	if err != nil {
+	if errors.Is(err, remote.ErrReceiverExists) {
```

Comment:
Does "Is" work correctly even if it's a longer string with ErrReceiverExists inside it?

Reply:
It does because we're wrapping the error using the %w directive. The test covers this behaviour